### PR TITLE
Feature/225 display dvh lvh

### LINKED ISSUE #225: Display DVH and LVH once plan execution is completed

### PR DESCRIPTION
**Pull request by ljelen**
closes #225

### CODE REVIEW

**Comment by ljelen** [comment]
Preview picture 

image

File: pytripgui/view/chart_widget.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    def title(self, title):
+        self.model.setTitle(title)
+
+    def add_series(self, x, y, name):
+        series = QLineSeries()
+        series.setName(name)
+
+        let_chart = np.vstack((x, y)).T
+
+        for x, y in let_chart:
```
Comment (grzanka) [inline pytripgui/view/chart_widget.py:38]: why not ue `zip`:

```
for x,y in zip(xserie,yserie):
#    dos th with x,y`
```

File: pytripgui/app_logic/chars.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+    def set_simulation_result(self, simulation_result):
+        self._charts = list()
+
+        for histogram_name in simulation_result.volume_histograms:
+            widget = ChartWidget()
+            widget.title = histogram_name
+
+            histogram = simulation_result.volume_histograms[histogram_name]
+            for organ_name in histogram:
```
Comment (grzanka) [inline pytripgui/app_logic/chars.py:22]: improve looping technique:

```
for name, value in some_dict.items():
    widget.add...(value.x, name)
```

File: pytripgui/view/chart_widget.py [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+    @title.setter
+    def title(self, title):
+        self.model.setTitle(title)
+
+    def add_series(self, x, y, name):
+        series = QLineSeries()
+        series.setName(name)
+
+        for x, y in zip(x, y):
```
Comment (grzanka) [inline pytripgui/view/chart_widget.py:34]: It is bad idea to use same variable names for iteration as for arguments in `zip` function.
Better would be:
```
for xlist, ylist in zip(x,y):
    series.append(xlist, ylist)
```

**Review by grzanka — CHANGES_REQUESTED** [review]
Few minor comments

**Review by grzanka — APPROVED** [review]
Looks OK now

**Pull request merged**